packages/custom-functions-metadata/src/generate.ts

```diff
@@ -60,6 +60,10 @@ export interface IParseTreeResult {
   functions: IFunction[];
 }
 
+export interface IParseTreeResultIncludingFields extends IParseTreeResult{
```

Comment:
Is all the places that use `IParseTreeResult` now going to use `IParseTreeResultIncludingFields`? If so, we could just add this `fields` to `IParseTreeResult`

Reply:
> Is all the places that use `IParseTreeResult` now going to use `IParseTreeResultIncludingFields`? If so, we could just add this `fields` to `IParseTreeResult`

Ok, I will change it.